### PR TITLE
[docs-revamp] - Clean up External assets guide

### PR DESCRIPTION
**Pull request by erinkcochran87**
## Changelog

NOCHANGELOG

### CODE REVIEW

File: docs/docs-beta/docs/guides/external-assets.md [hunk 1]
```diff
@@ -19,48 +23,54 @@ Examples of external assets could be files in a data lake that are populated by
...
-- A basic understanding of Dagster and assets. See the [Quick Start](/getting-started/quickstart) tutorial for an overview.
+- Familiarity with [Assets](/guides/data-assets)
 - Familiarity with [Sensors](/guides/sensors)
 </details>
 
----
+## Defining external assets
 
-## Creating and depending on external assets
+Let's say you have a partner sends you raw transaction data by SFTP on an almost daily basis. This data is later cleaned and stored in an internal data lake.
```
Comment (cmpadden) [inline docs/docs-beta/docs/guides/external-assets.md:37]:
```suggestion
Let's say you have a partner who sends you raw transaction data by SFTP on an almost daily basis. This data is later cleaned and stored in an internal data lake.
```

File: docs/docs-beta/docs/guides/external-assets.md [hunk 2]
```diff
@@ -19,48 +23,54 @@ Examples of external assets could be files in a data lake that are populated by
...
+<CodeExample filePath="guides/data-modeling/external-assets/pulling-with-sensors.py" language="python" />
 
-See the [sensors guide](/guides/sensors) for more information about sensors.
+Refer to the [Sensors guide](/guides/sensors) for more information about sensors.
 
-### "Pushing" with the REST API
+### Pushing with the REST API
 
-You can inform Dagster that an external asset has materialized by "pushing" the event from an external system to the REST API.
+You can inform Dagster that an external asset has materialized by pushing the event from an external system to the REST API. The following examples demonstrate how to inform Dagster that a materialzation of the `raw_transactions` external asset has occurred.
```
Comment (cmpadden) [inline docs/docs-beta/docs/guides/external-assets.md:66]:
```suggestion
You can inform Dagster that an external asset has materialized by pushing the event from an external system to the REST API. The following examples demonstrate how to inform Dagster that a materialization of the `raw_transactions` external asset has occurred.
```

**Review by cmpadden — APPROVED** [review]
Approved with 2 small fixes.

File: docs/docs-beta/docs/guides/external-assets.md (hunk 1 above)
Comment (erinkcochran87) [reply docs/docs-beta/docs/guides/external-assets.md:37]: I guess Vale doesn't fix everything 🤦‍♀️  ty!

File: docs/docs-beta/docs/guides/external-assets.md (hunk 2 above)
Comment (erinkcochran87) [reply docs/docs-beta/docs/guides/external-assets.md:66]: 🤦‍♀️

File: docs/docs-beta/docs/guides/external-assets.md [hunk 3]
```diff
@@ -1,16 +1,20 @@
...
+How are external assets different than native Dagster assets? The answer: Dagster can't materialize external assets directly or put them in a schedule. In these cases, an external system must inform Dagster when an external asset is updated.
 
-Examples of external assets could be files in a data lake that are populated by a bespoke internal tool, a CSV file delivered daily by SFTP from a partner, or a table in a data warehouse populated by another orchestrator.
+For example, external assets could be:
+
+- Files in a data lake that are populated by a bespoke internal tool
+- A CSV file delivered daily by SFTP from a partner
+- A table in a data warehouse populated by another orchestrator
 
 ## What you'll learn
```
Comment (cmpadden) [inline docs/docs-beta/docs/guides/external-assets.md:19]: Missed this originally, @erinkcochran87 , but we will want to remove this section.

File: docs/docs-beta/docs/guides/external-assets.md [hunk 4]
```diff
@@ -1,16 +1,20 @@
...
+One of Dagster's goals is to present a single unified lineage of all of the data assets in an organization, even if those assets are orchestrated by systems other than Dagster.
 
-**External assets** enable you to model assets orchestrated by other systems natively within Dagster's Asset catalog, and create new data assets downstream of these external assets.
+With **external assets**, you can model assets orchestrated by other systems natively within Dagster, ensuring you have a comprehensive catalog of your organization's data. You can also create new data assets downstream of these external assets.
 
-External assets differ from native Dagster assets in that Dagster can't materialize them directly or put them on a schedule. Instead, an external system must inform Dagster of when an external asset is updated.
+How are external assets different than native Dagster assets? The answer: Dagster can't materialize external assets directly or put them in a schedule. In these cases, an external system must inform Dagster when an external asset is updated.
```
[the remark this answers is not in the document]
Comment (erinkcochran87) [reply docs/docs-beta/docs/guides/external-assets.md]: Updated

**Pull request merged**